docs/content/build-and-deploy.md

```diff
@@ -60,10 +60,11 @@ Use the [Inspector](https://docs.improbable.io/reference/latest/shared/operate/i
 
 ### Cloud deployment
 To start a cloud deployment, follow these steps: 
-1. Upload your server-worker and client-worker assemblies. To do this, in a terminal window, from any directory, run `spatial cloud upload <assembly-name>` from the [`spatial` CLI](https://docs.improbable.io/reference/latest/shared/glossary#the-spatial-command-line-tool-cli). 
+1. Ensure you set the `name` field in the `spatialos.json` to a SpatialOS project that you have access to. You can find your project name in the [SpatialOS Console](https://console.improbable.io/). It’ll be something like `beta_someword_anotherword_000`.
```

Comment:
Where can I find the `spatialos.json` file? 
(Also, I would put the context a the beginning of the sentence to help orientate the user. )
Also, is this just any SpatialOS project? Or must it be a specific one? How do I get access to the project? Is there somewhere we have this documented? Should this be a step before this?


1. Specify the SpatialOS project name you want to deploy to. To do this, locate the `spatialos.json` file in the <where is it?!> and edit the file so the `name` field matches your SpatialOS project name. You can find your project name in the [SpatialOS Console](https://console.improbable.io/). It’ll be something like `beta_someword_anotherword_000`. (To set up your project in SpatialOS, follow the instructions <somewhere?!>.

Reply:
yeah, they should have a SpatialOS account. not sure whether we say that anywhere? should probably happen during the setup & installation guide

Reply:
I've added "signup for a SpatialOS account" in the setup guide revamp branch